EvolvSwiftSDK/EvolvSwiftSDK/Sources/Core/EvolvContextImpl.swift

```diff
@@ -19,44 +19,32 @@
 
 import Foundation
 
-public class EvolvContextImpl: EvolvContext {
-    
-    private var uid: String
-    private var sid: String?
+public struct EvolvContextImpl: EvolvContext {
     private var remoteContext: [String : Any] = [:]
     private var localContext: [String : Any] = [:]
-    private var initialized = false
-    
     
-    private var evolvConfig: EvolvConfigImpl?
-    private var evolvStore: EvolvStoreImpl?
+    public var configuration: Configuration?
     
-    public required init(uid: String, remoteContext: [String : Any], localContext: [String : Any]) {
-        
-        self.uid = uid
+    public init(remoteContext: [String : Any], localContext: [String : Any]) {
         self.localContext = localContext
         self.remoteContext = remoteContext
     }
     
     public func resolve() -> [String: Any] {
-        var effectiveContext: [String: Any] = [:]
-        return effectiveContext
+        return [:]
     }
     
-    public func set(key: String, value: String, local: Bool = false) -> [String : Any] {
-        
-        var context = local ? localContext : remoteContext
-        context.updateValue(value, forKey: key)
-        return context
+    public mutating func set(key: String, value: Any, local: Bool = false) -> [String : Any] {
+        if local {
```

Comment:
the context are general objects that can have a nested structure. e.g
{
   location: {
      country: 'USA'
   },
   device: mobile
}

These contexts look like they're arrays - do we need to update the structure here to support nesting?

Reply:
No, the context structures here are dictionaries, that can hold values of any type, including nested structures.

Reply:
I noticed a different problem here: we do not expose function for mutation of the context after initialisation, I'll address that in the future PR.